bitnami/grafana/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+name: grafana
+version: 0.1.0
+appVersion: "6.3.3"
+description: Grafana is an open source metric analytics and visualization suite.
```

Comment:
Use the description available at https://github.com/bitnami/bitnami-docker-grafana#what-is-grafana